src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
Yes, that's how I understood your reasoning. That they look the same, but the virtual light source has changed. The scenario I'm thinking about is the possible invention of `Rgb<SrgbPrim<D65>, D50>` (or any other combination of existing parts) and how to express it as a type. I think that we should leave as many doors open as possible for the users to extend the set of colors. We can't encode every single possible combination, but we can allow the users to add the ones they may need.